packages/conda-store/package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "@mamba-org/conda-store",
+  "version": "0.1.0",
+  "description": "A JupyterLab extension.",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/mamba-org/gator",
+  "bugs": {
+    "url": "https://github.com/mamba-org/gator/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Peyton Murray",
+    "email": "pdmurray@quansight.com"
+  },
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
+    "schema/*.json"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mamba-org/gator.git"
+  },
+  "scripts": {
+    "build": "tsc --project .",
+    "build:dev": "yarn run build",
+    "build:prod": "yarn run build",
+    "clean": "rimraf lib tsconfig.tsbuildinfo",
+    "lab:link": "jupyter labextension link . --no-build",
+    "prepare": "yarn run clean && yarn run build",
+    "test": "jest --coverage",
```

@peytondmurray could you apply the following suggestion to correct the CI?

```suggestion
    "test": "echo no jest tests",
```

---

For sure!